pom.xml

```diff
@@ -1530,6 +1550,7 @@
                 <spark.major.version>spark2.2</spark.major.version>
                 <zookeeper.version>3.4.5-cdh5.14.0</zookeeper.version>
                 <spark.version>2.2.2</spark.version>
+                <scala.version>2.11.8</scala.version>
```

Comment:
Why is this 2.11.8 while the others are 2.11.6? Why so many duplicates? Why not to default to 2.11.6 globally and overwrite with a custom value per profile?